README.md

```diff
@@ -1 +1,226 @@
-# The_Future_Mobility_BackEnd
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
According to the requirements for 3-4 members:

![image](https://user-images.githubusercontent.com/57421823/213175440-ce68d1fd-69cc-4ac8-b7e8-249612c05d2e.png)

I couldn't add any docs.

Therefore:

- [ ] Kindly add full documentation for your API, so we can meet this requirement.  If docs exist, kindly add a section on your readme about how to access them.

That way, your API will be well documented, and all requirements will be met.  I suggest using swagger because it's simple and powerful. 💪

Reply:
The link to API documentation is already present on Readme.md:

![image](https://user-images.githubusercontent.com/55507343/213192514-ccff49ed-4446-436b-8ef5-1b4399d37abd.png)

Reply:
@wandergithub Thanks for pointing this out.  It will be taken into account. 👍